python/src/QuantileMatchingFactory_doc.i.in

```diff
@@ -0,0 +1,227 @@
+%feature("docstring") OT::QuantileMatchingFactory
+"Estimation by matching quantiles.
+
+Implements generic estimation by matching quantiles.
+
+Parameters
+----------
+distribution : :class:`~openturns.Distribution`
+    The distribution defining the parametric model to be adjusted to data.
+    Its parameters define the starting point of the algorithm.
+probabilities : sequence of float, optional
+    The probabilities :math:`p_i \in [0,1]` corresponding to the quantiles.
+    The default value of the list of probabilities defined as follows.
+    We define the interval :math:`[\epsilon, 1 - \epsilon]` where the value of
+    :math:`\epsilon` is defined by the `QuantileMatchingFactory-QuantileEpsilon` key of the 
+    `:class:`~openturns.ResourceMap`.
```

Comment:
This produces:
![image](https://user-images.githubusercontent.com/31351465/190729045-0397abe5-b545-4cdd-8aab-f9ec85b5f569.png)

```suggestion
    :class:`~openturns.ResourceMap`.
```